src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -101,6 +101,12 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// <remarks>This property is used only for unit testing.</remarks>
         internal IConfigurationClientManager ClientManager { get; set; }
 
+        /// <summary>
+        /// An optional timespan value to set the minimum backoff duration to a value other than the default.
+        /// </summary>
+        /// <remarks>This property is used only for unit testing.</remarks>
```

Comment:
This property is not only used for unit testing. It's referenced in this file for backoff. It would be appropriate to say this property is internal for availability in unit tests. But even that seems unnecessary.

Reply:
That makes sense, I'll remove the remarks.